auditbeat/module/auditd/_meta/docs.asciidoc

```diff
@@ -158,6 +161,10 @@ Defining any audit rules in the config causes {beatname_uc} to purge all
 existing audit rules prior to adding the rules specified in the config.
 Therefore it is unnecessary and unsupported to include a `-D` (delete all) rule.
 
+The rules declared in the `audit_rules` field are loaded first. Then the files
```

Comment:
I think I would move this above with the reference information under `audit_rule_files`.